grid-demo/src/app/angular/column-dom-performance/column-dom-performance.component.ts

```diff
@@ -10,7 +10,7 @@ import { Vm, generateVms, columns } from './../../data/vm.generator';
 })
 export class AngularColumnDomPerformanceComponent {
   readonly vms: Observable<Vm[]>;
-  readonly columns = columns.map(column => ({ ...column, hidden: false }));
+  readonly columns = columns;
```

Comment:
Should we use `[...columns]` here to prevent hidden column state from be shared between demos? Or do we want that to be shared?

Reply:
I think you're right. I'll change it